ApollosProject/apollos-apps
# feat: Pulse Image

## ✏️ Description
There is a new UI Kit component that we need to introduce in order to launch Daily Habits: PulseImage.

For reference, check out the [Figma Component](https://www.figma.com/file/OvEmzj97LwXzL0E5yBTvfB/pulse-image?node-id=4459%3A68805) and associated documentation. The pulse image can be used with 2 or 3 layers of an image. By default, there is a gradient overlay that is repeated on each layer, but you are free to change/negate this completely. It's preferred to use aspect ratios that are square or portrait. Landscape images are not advised. The default width is 58px, but that can be adjusted as need be.

## 🔬 To Test
1. Set the `PROJECT_VARIANT` to `storybook`
2. `yarn && yarn start` and `yarn ios`
3. Check out the `PulseImage` stories

## 📸 Screenshots
| Feature | Simulator | Figma | Links | Design Considerations |
| --- | --- | --- | --- | ---|
| Default Rendering | ![CleanShot 2022-11-08 at 15 05 23](https://user-images.githubusercontent.com/45076058/200666055-bcfd11ac-6960-4226-83d1-05c7643673f8.png) | ![CleanShot 2022-11-08 at 15 17 04](https://user-images.githubusercontent.com/45076058/200666447-4079be0e-8702-4b8e-976d-453ad7ab87b0.png) | [Link to Figma](https://www.figma.com/file/OvEmzj97LwXzL0E5yBTvfB/pulse-image?node-id=4459%3A68805) | n/a |
| Displaying an Icon | ![CleanShot 2022-11-08 at 15 05 10](https://user-images.githubusercontent.com/45076058/200666644-e721ea85-d7fa-4e46-839e-5f3e16c07784.png) | ![CleanShot 2022-11-08 at 15 21 18](https://user-images.githubusercontent.com/45076058/200667221-9f2a5043-83df-4fc4-b922-810d92648fa2.png) | n/a | n/a |
| Color Adjustments | ![CleanShot 2022-11-08 at 15 04 57](https://user-images.githubusercontent.com/45076058/200667317-eb26ab6c-6d91-4b68-9a41-af1a4222da08.png) | ![CleanShot 2022-11-08 at 15 27 47](https://user-images.githubusercontent.com/45076058/200668497-cc4aa7a5-4f17-497b-bdf3-711d2a3bd31b.png) | n/a | n/a |

## ⚠️ To-do Before Merge

- [ ] [This branch](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/branch/OvEmzj97LwXzL0E5yBTvfB/UI-Kit-2.0?node-id=2605%3A13981) in Figma needs to be merged into the main UI Kit (@conrad-vanl idk what the processor for reviews/approvals looks like for that)

## 📝 Checklist
